groovy/groovy.kit/nbproject/project.xml

```diff
@@ -38,18 +38,6 @@
                         <specification-version>1.10</specification-version>
                     </run-dependency>
                 </dependency>
-                <dependency>
-                    <code-name-base>org.netbeans.modules.groovy.grails</code-name-base>
-                    <run-dependency>
-                        <specification-version>1.7</specification-version>
-                    </run-dependency>
-                </dependency>
-                <dependency>
-                    <code-name-base>org.netbeans.modules.groovy.grailsproject</code-name-base>
-                    <run-dependency>
-                        <specification-version>1.9</specification-version>
-                    </run-dependency>
-                </dependency>
```

Comment:
It would be good if the PRs don't get mixed. This belongs into the groovy activation PR if I'm not mistaken.

Reply:
I thought this was coupled to the upgrade... but I can revert, no problem.